Parsedown.php

```diff
@@ -216,7 +207,7 @@ protected function linesElements(array $lines)
 
             if (isset($CurrentBlock['continuable']))
             {
-                $Block = $this->{'block'.$CurrentBlock['type'].'Continue'}($Line, $CurrentBlock);
+                $Block = $this->{"block{$CurrentBlock['type']}Continue"}($Line, $CurrentBlock);
```

Comment:
readability:
```php
$methodName = 'block' . $CurrentBlock['type'] . 'Continue';
$Block = $this->$methodName($Line, $CurrentBlock);
```

Reply:
👍 

As mentioned in:
> Adjustments to string concatenation syntax appear to offer no observable difference in Parsedown in my testing (despite historic benchmarks from the PHP community apparently indicating a difference).

I think I'll probably change this back to single quotes (as you've mentioned for readability), and since it doesn't offer any observable performance advantage in my testing. (I recall a long time ago that using `.` over interpolation had an observable negative difference – I guess things have changed now.)

Reply:
I don't really aim for string interpolation vs. string concatenation here, but rather for defining a complex method name separately.

I don't think that there's a "real" performance difference between string interpolation and string concatenation. I personally prefer string concatenation because I think it's way more readable, especially for complex strings. However, others might say the complete different, so this is basically just a preference.